improver/psychrometric_calculations/psychrometric_calculations.py

```diff
@@ -843,11 +843,16 @@ def find_max_in_nbhood_orography(self, orography_cube):
                 The cube containing the maximum in a neighbourhood of the
                 orography data.
         """
-        radius_in_metres = number_of_grid_cells_to_distance(
-            orography_cube, self.grid_point_radius
-        )
-        max_in_nbhood_orog = OccurrenceWithinVicinity(radius_in_metres)(orography_cube)
-        return max_in_nbhood_orog
+        if self.grid_point_radius >= 1:
```

Comment:
Could the docstring above be updated? Something like: 
```
        Find the maximum value of the orography in the neighbourhood around 
        each grid point. If the grid point radius is zero, the orography is used 
        directly without finding the maximum height in a neighbourhood.
```
Note that the `Returns` section could also be updated.

Reply:
Done